build-and-run.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+
+printfn() {
+	printf "$1\n"
+}
+
+# get arguments
+while getopts i:e:d:h:w:o:r:s:B:m:tlb flag; do
+	case "$flag" in
+	i) projectFolder=${OPTARG} ;;
+	e) exportCount=${OPTARG} ;;
+	d) delayms=${OPTARG} ;;
+	s) totalTime=${OPTARG} ;;
+	h) renderHeight=${OPTARG} ;;
+	w) renderWidth=${OPTARG} ;;
+	r) renderPath=${OPTARG} ;;
+	o) jsonFolder=${OPTARG} ;;
+	t) transmit="--transmit" ;;
+	l) logJson="--logExport" ;;
+	b) build=true ;;
+	B) buildDir=${OPTARG} ;;
+	m) copyProject=${OPTARG} ;;
+	\?)
+		printfn "Invalid arg ${OPTARG}" >&2
+		exit 1
+		;;
+	esac
+done
+
+# validate project folder
+if [ ! -d "$projectFolder" ] && [ ! -r "$projectFolder" ]; then
```

Comment:
Is this supposed to be an 'or' condition?

Reply:
Fixed in #35.